### PR TITLE
Add support for setting the parent cgroup (added in Docker API 1.18)

### PR DESCRIPTION
**Pull request by guenter**

### CODE REVIEW

File: container.go [hunk 1]
```diff
@@ -418,6 +418,7 @@ type HostConfig struct {
 	LogConfig       LogConfig              `json:"LogConfig,omitempty" yaml:"LogConfig,omitempty"`
 	ReadonlyRootfs  bool                   `json:"ReadonlyRootfs,omitempty" yaml:"ReadonlyRootfs,omitempty"`
 	SecurityOpt     []string               `json:"SecurityOpt,omitempty" yaml:"SecurityOpt,omitempty"`
+	CgroupParent    string                 `json:"CgroupParent,omitempty" yaml:"CgroupParent,omitempty"` // For Docker API v1.18 and above only
```
Comment (fsouza) [inline container.go]: Please remove this comment.

**Comment by fsouza** [comment]
Hi @guenter, thanks for contributing! The change looks good to me, but I'd prefer to remove that comment.

**Comment by guenter** [comment]
@fsouza thanks! Removed.

**Comment by fsouza** [comment]
Thank you!

**Pull request merged**